Makefile

```diff
@@ -126,6 +126,7 @@ test-cmd: build
 # Example:
 #   make test-end-to-end
 test-end-to-end: build
+	hack/env hack/verify-generated-protobuf.sh # Test the protobuf serializations when we know Docker is available
```

Comment:
> It does not check upstream protobuf, what do you mean by "additional
> checks"?

We had a couple API carries since shipped a `v1` first.  We need to make sure that never happens again or if it does, make sure that its not serialized to protobuf.

I guess we could all promise to do no evil.